index.js

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (c) 2014, Yahoo Inc. All rights reserved.
```

Comment:
Hi Eric,
Here's the link I was using for copyright date guidelines.https://yahoo.jiveon.com/groups/oswg/blog/2014/12/15/copyright-year-update-reminder

Jeff
P.S. Is it true that I'm not supposed to include internal URLs such as above in github comments?

      From: Eric Ferraiuolo <notifications@github.com>
 To: yahoo/express-state <express-state@noreply.github.com> 
Cc: Jeff Conniff <jconniff@yahoo-inc.com> 
 Sent: Thursday, February 26, 2015 6:17 PM
 Subject: Re: [express-state] adding/updating license info (#36)
   
In index.js:> @@ -1,3 +1,9 @@
> +/*
> + * Copyright (c) 2014, Yahoo Inc. All rights reserved.
@jconniff was this left at 2014 because the file hasn't changed? Whereas the others are now 2015?—
Reply to this email directly or view it on GitHub.